modules/ROOT/pages/telemetry-exporter.adoc

```diff
@@ -0,0 +1,145 @@
+
+include::partial$include-nav-steps-telemetry-permissions.adoc[connectionPermission]
+
+include::partial$include-nav-telemetry-exporter.adoc[]
+
+. Click the *Connections* tab.
+. Click *New Connection*.
++
+The *Create New Connection* interface appears, showing the *Connection Details* page.
+. In the *Connection Details* page, enter the following information: 
+* *Connection Name*: A name for your connection using only alphanumeric characters.
+* *Destination Type*: The third-party app to which you want to transfer data. 
+* *Endpoint URL*: The endpoint that enables external connections to your third-party app. The location of this endpoint depends on your app.
+. Click *Next*.
+. On the *Authentication* page, enter the information needed to authenticate your app. Some apps need only access tokens, while others also require a header key.
+. Click *Test Connection* to ensure Anypoint Platform can connect to your third-party app.
+. Click *Save Connection*. 
++
+The new connection appears in the *Connections* tab on the *Telemetry Exporter* page.
+
+=== Edit a Connection
+
+include::partial$include-nav-steps-telemetry-permissions.adoc[tag=connectionPermission]
+
+include::partial$include-nav-telemetry-exporter.adoc[]
+
+. Click the *Connections* tab.
+. Next to the connection you want to edit, click the *...* menu and select *Edit Connection*.
++
+The *Edit Connection* interface appears, showing the *Connection Details* page. 
+
+. In the *Connection Details* page, edit each field you want to change, and then click *Next.*
+. In the *Authentication* page, edit each field you want to change, then click *Test Connection*. When the connection is successful, click *Save Connection*.
+
+=== Delete a Connection
+
+include::partial$include-nav-steps-telemetry-permissions.adoc[tag=connectionPermission]
+
+include::partial$include-nav-telemetry-exporter.adoc[]
+
+. Next to the connection you want to delete, click the *...* menu and select *Delete Connection*.
+. A warning appears:
+* If the connection is not in use, the warning confirms you want to delete the connection. Click *Delete Connection*.
+* If the connection is in use, the warning states you must delete or redirect connections for any configurations that use the connection. Click *Dismiss*, and then <<edit-configuration,edit your configurations>> to use other existing connections before deleting the current connection.
+
+== Manage Configurations
+
+After you create a connection to a third-party app, you must create a _configuration_ to specify what kind of data to send to your third-party app, as well as the frequency of data transfers. When you create a new configuration, you specify a business group, environment type, and a destination.
+
+The Telemetry Exporter feature runs a service every hour to apply the changes for the configurations and connections, so once the configuration is created, it can take up to an hour for the configuration to be applied and start exporting data. 
+
```

Comment:
You need the Telemetry Exporter Configuration Manager permission to manage configurations.